src/backend/src/services/tasks.services.ts

```diff
@@ -161,10 +166,12 @@ export default class TasksService {
     // this throws if any of the users aren't found
     const assigneeUsers = await getUsers(assignees);
 
-    const team = originalTask.wbsElement?.project?.team;
-    if (!team) throw new HttpException(400, 'This project needs to be assigned to a team to create a task!');
-    if (!allUsersOnTeam(team, assigneeUsers)) {
-      throw new HttpException(400, `All assignees must be part of the project's team!`);
+    const teams = originalTask.wbsElement?.project?.teams;
+    if (!teams) throw new HttpException(400, 'This project needs to be assigned to a team to create a task!');
+
+    // checks if there is a user that does not belong on any team of the project
+    if (assigneeUsers.map((user) => teams.map((team) => isUserOnTeam(team, user)).includes(true)).includes(false)) {
```

Comment:
I think this is the logic u want in the create. I'd suggest putting it in a helper method in the utils file, and maybe even break it into two helper methods (isUserOnOneOfTeams, areUsersOnOneOfTeams or smthn) and use it in both places